lnrpc/walletrpc/config_active.go

```diff
@@ -76,4 +78,9 @@ type Config struct {
 	// CoinSelectionStrategy is the strategy that is used for selecting
 	// coins when funding a transaction.
 	CoinSelectionStrategy wallet.CoinSelectionStrategy
+
+	// GetWaitingCloseChannel returns the waiting close channel for a
+	// specified channel point.
+	GetWaitingCloseChannel func(chanPoint wire.OutPoint) (
```

Comment:
hmm think we should avoid passing functions as config values, think `CurrentNumAnchorChans` was a bad start.

Reply:
changed, but did not touch the `CurrentNumAnchorChans ` should probably refactored in a separate PR.